homeassistant/components/edl21/config_flow.py

```diff
@@ -0,0 +1,68 @@
+"""Config flow for EDL21 integration."""
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_SERIAL_PORT, DEFAULT_DEVICE_NAME, DOMAIN
+
+
+class EDL21ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """EDL21 config flow."""
+
+    VERSION = 1
+
+    async def async_step_import(self, import_config: dict[str, Any]) -> FlowResult:
+        """Import a config entry from configuration.yaml."""
+
+        self._async_abort_entries_match(
+            {
+                CONF_SERIAL_PORT: import_config[CONF_SERIAL_PORT],
+            }
```

Comment:
Style: does it not fit on one line?
```suggestion
            {CONF_SERIAL_PORT: import_config[CONF_SERIAL_PORT]}
```